src/components/CodeView/utils.tsx

```diff
@@ -36,3 +36,17 @@ export const mapWithDepth = (depth: number, _mapChild = mapChild) => {
 export const getLines = (content: string) => {
   return content.replace('\r\n', '\n').split('\n');
 };
+
```

Comment:
I don't think the two comment blocks below are useful. They both paraphrase the function names.